osmaxx/excerptexport/templates/excerptexport/base.html

```diff
@@ -12,6 +11,15 @@
 {% endblock %}
 
 {% block scripts %}
-<!-- Helpers -->
-<script type="text/javascript" src="{% static 'excerptexport/scripts/helperFunctions.js' %}"></script>
-{% endblock %}
\ No newline at end of file
+    <!-- Helpers -->
+    <script type="text/javascript" src="{% static 'excerptexport/scripts/helperFunctions.js' %}"></script>
+
+    {# MAP SCRIPTS #}
```

Comment:
Because I copied the old code. They should probably all be template comments, they are only developer comments.